ci/install-windows.ps1

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env powershell
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Stop on errors, similar (but not quite as useful as) "set -e" on Unix shells ...
```

Comment:
Please change the `...` comments into simple statements.  e.g.:

```
# Stop on errors.  This has a similar effect as `set -e` on Unix shells.
```

Reply:
And I know it is my fault, I used `...` comments in other project.  We settled on assertion-type comments for **this** project though.

Reply:
Done.